### DISCUSSION TITLE
Le bandeau de consentement Piwik Pro toujours actif

### DISCUSSION THREAD

**Discussion by philmarty** [AmauriC/tarteaucitron.js#1117] [Q&A]
Bonjour,
Nous avons ajouté un appel à Piwik pro via le script Tarte au citron, il n'y a pas de code html de Piwik dnas nos pages.
Lorsque l'on accepte les cookies de Piwik via la boite de dialogue Tarte au citron, la boite de dialogue de consentement Piwik apparaît )à son tour, lorsque l'on change de page.
Comment faire pour supprimer cette boite de consentement Piwik puisque les cookies sont déjà accepté pour ce service ?
Le problème est visible sur ce site : https://www.wesireport.com/
Merci par avance !

**Comment by AmauriC** [comment]
Bonjour,

Je viens de tester mais je ne vois pas de dialogue piwik.

Dans tous les cas ce n'est pas chargé par tarteaucitron, très probablement un réglage à faire depuis l'interface piwik ; )

**Comment by philmarty** [comment]
Curieux que tu ne le vois pas, je viens de tester sur mon tel aussi, le
comportement est le même.
J'arrive sur le site wesireport.com, le popup tarteaucitron s"ouvre,
j'accepte tout. Tout se passe bien.
Lorsque je change de page, un beandeau de consentement Piwik apparaît.

Pour info, j'ai juste mis la ligne "(tarteaucitron.job = tarteaucitron.job
|| []).push('piwikpro');" et pas la ligne précédente qui envoie l'ID, car
cela est géré par Google Tag Manager.


Le lun. 12 juin 2023 à 17:06, Amauri CHAMPEAUX ***@***.***> a
écrit :

**Reply by AmauriC** [reply]
Ok, je vois maintenant.

C'est bien quelque chose côté piwik, voir avec leur assistance comment désactiver ce double bandeau, ce n'est pas côté tarteaucitron 🤷‍♂️

**Reply by AmauriC** [reply]
https://help.piwik.pro/support/privacy/setting-consent-manager/

**Reply by philmarty** [reply]
Ok, on regarde, merci beaucoup pour la réponse rapide !

**Reply by philmarty** [reply]
Ok, c'est réglé, c'était bien ce réglage à modifier.
Encore merci.
